packages/flutter_tools/test/integration/hot_reload_test.dart

```diff
@@ -44,12 +42,10 @@ void main() {
           new Uri.file(_project.breakpointFile).toString(),
           _project.breakpointLine);
       expect(isolate.pauseEvent, isInstanceOf<VMPauseBreakpointEvent>());
-      // TODO(dantup): Unskip for Mac when [1] is fixed, unskip on Windows when
-      // both [1] and [2] are fixed.
+
+      // TODO(dantup): Unskip for Mac when [1] is fixed.
       // [1] hot reload/breakpoints fail when uris prefixed with file://
       //     https://github.com/flutter/flutter/issues/18441
-      // [2] flutter-tester doesn't work on Windows
-      //     https://github.com/flutter/flutter/issues/17833
-    }, skip: !platform.isLinux);
+    }, skip: !platform.isLinux && !platform.isWindows);
```

Comment:
Yeah, something is up with the mac, not sure what. But #18441 seems to be no longer a problem - uris should work.

Reply:
https://github.com/flutter/flutter/issues/18441 still seems an issue on my Mac (unskipping this test fails waiting for an isolate pause event). I'm not sure if it's just started passing on Windows, or my report for this issue was just bad. I'll add a note to that issue that it may be macOS only anyway.